runtime/compiler/control/HookedByTheJit.cpp

```diff
@@ -612,6 +612,15 @@ static void jitHookInitializeSendTarget(J9HookInterface * * hook, UDATA eventNum
       TR_VerboseLog::writeLineLocked(TR_Vlog_PERF, "Setting count=%d for %s", count, buffer);
       }
 
+#if defined(J9VM_OPT_MICROJIT)
+   if(optionsJIT->_mjitEnabled)
+      {
+      int32_t mjitCount = optionsJIT->_mjitInitialCount;
+      if(count)
+         TR::CompilationInfo::setInitialMJITCountUnsynchronized(method,mjitCount,count);
+      }
+#endif
+
```

Comment:
I wonder if we should keep the low counts when methods are found in the SCC and can be quickly loaded from there. As it stands, the microjit code is oblivious of the SCC/AOT

Reply:
This think this would be a good thing to discuss at the next MicroJIT community call (how we can be more aware of the other JIT options and facilities that have an impact on MicroJIT).

Reply:
Also, if we have AOT enabled and there is a method in the SCC as indicated by this code, we may want to load that AOT body instead of compiling with microjit. This may happen anyway (I need to follow the code)

Reply:
Regarding the counts, say the user has specified very low counts for a method (or all) with `-Xjit:count=`. The microjit counts could be higher and against the intent of the user. I think we should take the minimum between the JIT/AOT counts and microjit counts